bootstrap.sh

```diff
@@ -10,7 +10,7 @@ CURRENT_DIR=$(pwd)
 # default to clang
 CXX=${CXX:-clang++}
 TARGET=${TARGET:-Release}
-OSRM_RELEASE=${OSRM_RELEASE:-"develop"}
+OSRM_RELEASE=${OSRM_RELEASE:-"rewrite/new-api"}
```

Comment:
@lbud do you think we can get @MoKob's guidance work into the pre-release as well? He has this PR up https://github.com/Project-OSRM/osrm-backend/pull/2007 which he rebases every day against the `rewrite/new-api` branch by @TheMarex